### PR TITLE
fix: attempts to fix the font errors on Travis without removing the f…

### PR DESCRIPTION
**Pull request by colincclark**
Trying to remove the need for fetch-font script on Travis (CI). Upon removing the call to this script, the fonts disappeared on staging! This is a first-pass hail-mary attempt to fix it. Can only be tested on Travis and staging, as these scripts only run on CI during a master deploy. I amended fonts related to the article story, so can be tested there

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/68097d39234b5b90a1581186489d51225eac8cd7/dextrose-test/index.html

**Comment by colincclark** [comment]
@craigbilner amended to only test one font as per suggestion

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/483509f08f65c6132938ec22dfd1fa3b9356b877/dextrose-test/index.html

File: .storybook/preview-head.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
   @font-face {
     font-family: TimesModern-Regular;
-    src: url(/fonts/TimesModern-Regular.woff2) format("woff2"), url(/fonts/TimesModern-Regular.woff) format("woff"), url(/fonts/TimesModern-Regular.ttf) format("truetype"), url(
+    src: url(https://www.thetimes.co.uk/d/fonts/TimesModern-Regular.woff2) format("woff2"), url(https://www.thetimes.co.uk/d/fonts/TimesModern-Regular.woff) format("woff"), url(https://www.thetimes.co.uk/d/fonts/TimesModern-Regular.ttf) format("truetype"), url(
```
Comment (toboid) [inline .storybook/preview-head.html]: 🗣  as discussed, website only serves fonts with a hash in the file name to facilitate long cache times

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/cbbfbcd91d22074b38e44a5df3b968a8278aacd5/dextrose-test/index.html

**Comment by aronshamash** [comment]
Think it is ready to give it a go

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/435ced1046c63091f2298c99636066d856ac11cc/dextrose-test/index.html

**Pull request merged**